File: 4.study/7.Logcal/1.old/AND/review/1-1.rb
# AND를 사용한 예제 복습
puts("id를 입력해주세요.\n")
input_id = gets.chomp()
puts("비밀번호를 입력해주세요.\n")
input_pwd = gets.chomp()
real_id = "gst"
real_pwd = "875"
if real_id == input_id and real_pwd ==input_pwd
    puts("Hello!")
else
    puts("잘못된 비밀번호입니다.")
end

File: 4.study/7.Logcal/1.old/AND/1-1.py
# AND를 사용한 예제
input_id = input("아이디를 입력해주세요.\n")
input_pwd = input("비밀번호를 입력해주세요.\n")
real_id = "gst"
real_pwd = "11"
if real_id == input_id and real_pwd == input_pwd:
    print("Hello!")
else:
    print("로그인에 실패했습니다.")
